libs/packages/components/tsconfig.lib.prod.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./tsconfig.lib.json",
+  "angularCompilerOptions": {
+    "enableIvy": false
```

Comment:
I've noticed Ivy has been disabled on all tsconfigs of our libraries. Were we running into issues when enabling it?